pkg/assets/asset.go

```diff
@@ -85,11 +85,12 @@ func (s *AssetsStore) loadaccountsStore(assetsJSONPath string) error {
 		}
 
 		for _, asset := range accountData.Assets {
+			decimals := asset.Decimals // Copy the decimals value to avoid a pointer to a local variable
 			assetInfo := models.AssetInfo{
 				Address:  asset.ContractAddress,
 				Name:     asset.Name,
 				Symbol:   asset.Symbol,
-				Decimals: &asset.Decimals,
+				Decimals: &decimals,
```

Comment:
are you sure you want to keep the `&`?

Reply:
yes because I need a pointer

Reply:
so I found your comment above 
> // Copy the decimals value to avoid a pointer to a local variable 

a bit misleading because it seems that you're exactly getting a pointer to a local variable. But maybe there's a Go trick, I'd like to know, here.